setup/mainmenu.c

```diff
@@ -182,6 +183,9 @@ static void MissionSet(void)
 void D_DoomMain(void)
 {
    //SetupMission(MissionSet);
+   /* The parameters are not used anyway */
+   atexit((void (*)(void))DoQuit);
```

Comment:
We don't need this line, I think it's better to add `SDL_Quit()` to the end of `StartGame` function.